docs/index.md

```diff
@@ -1,14 +1,27 @@
-# COB Ontology Documentation
+# The Core Ontology for Biology and Biomedicine
 
 [//]: # "This file is meant to be edited by the ontology maintainer."
 
-This project is an attempt to bring together key terms from a wide range of [Open Biological and Biomedical Ontology (OBO)](http://obofoundry.org) projects into a single, small ontology. The goal is to improve interoperabilty and reuse across the OBO community through better coordination of key terms. Our plan is to keep this ontology small, but ensure that one or more COB terms can be used as the root of any given OBO library ontology.
+\[ [Browse COB on Ontobee](https://ontobee.org/ontology/COB) \] \[ [View COB on GitHub](https://github.com/OBOFoundry/COB/) \]
 
-## Editors Guide
+The Core Ontology for Biology and Biomedicine (COB) aims to bring together key terms from a wide range of [Open Biological and Biomedical Ontologies (OBO)](http://obofoundry.org) Foundry projects into a single, small ontology. COB anchors these terms in the Basic Formal Ontology (BFO) which ensures that COB based ontologies are compatible with BFO, but COB elects to omit some of the complexities of BFO that have proven confusing to users in the biomedical domain. Our goal is to improve interoperability and term reuse across the OBO community through better coordination of widely-used terms. We intend to keep this ontology small while ensuring that one or more COB terms can be used as the root of any given OBO Foundry ontology.
 
-You can find descriptions of the standard ontology engineering workflows [here](odk-workflows/index.md).
+**The 2024 COB Workshop will be hosted on September 23–24, 2024 at the La Jolla Institute for Immunology in La Jolla, CA. Please see the [Workshop](workshop2024.md) page for more information.**
+
+## Why Use COB?
+COB is designed to serve as a bridge between domain ontologies. COB ensures interoperability between the ontologies that extend from it, many of which were developed independently of each other and would otherwise be difficult to use together. COB is also designed to be small and concise in order to minimize clutter and promote ease of use by non-ontologists.
```

Comment:
Is "bridge" the right analogy? Uberon is a bridge, I think COB should be described as a "top level" ontology for biology. It also does not "ensure interoperability" all by itself, but provides a framework for interoperable representations in domain ontologies.

"mimize clutter" is not a meaningful phrase to my ears

Reply:
So how about: "COB is designed as a top level ontology that provides a framework in which different domain ontologies can interoperate"

Reply:
Yes, perfect.

Reply:
Sounds good, I'll make changes accordingly.